gh-95380: Remove the 1024 bytes limit in fcntl.fcntl() and fcntl.ioctl() (#132907)
python/cpython: Misc/NEWS.d/next/Library/2025-04-25-11-53-37.gh-issue-95380.7dvPe-.rst

> Remove the 1024 bytes limit on the size of not mutated bytes-like argument
> in :func:`fcntl.fcntl` nad :func:`fcntl.ioctl`.

Reviewer: ```suggestion
:func:`fcntl.fcntl` and :func:`fcntl.ioctl`: Remove the 1024 bytes limit
on the size of not mutated bytes-like argument.
```
